Makefile

```diff
@@ -80,14 +81,19 @@ lint:
 	$(ISORT) --check-only --diff $(EXAMPLES_DIR) $(PACKAGE) $(SETUP_PY) $(TESTS_DIR)
 	check-manifest
 
-coverage:
+coverage-clean:
 	$(COVERAGE) erase
-	$(COVERAGE) run "--include=$(PACKAGE)/*.py,$(TESTS_DIR)/*.py" --branch -m unittest
-	$(COVERAGE) report "--include=$(PACKAGE)/*.py,$(TESTS_DIR)/*.py"
-	$(COVERAGE) html "--include=$(PACKAGE)/*.py,$(TESTS_DIR)/*.py"
+	@rm -rf .coverage htmlcov/
+
+coverage-test:
+	$(PYTHON_ERROR_ON_WARN) $(COVERAGE_PATH) run -m unittest
```

Comment:
[Continuing the conversation](https://github.com/FactoryBoy/factory_boy/pull/952/files#r908786905)

Reply:
@rbarrois

Reply:
@rbarrois friendly bump :)